src/ontology/edits/oeo-physical.omn

```diff
@@ -731,7 +767,11 @@ Class: OEO_00000027
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "Nitrous oxide is a portion of matter with the chemical formula N2O. It has a gaseous normal state of matter. It occurs naturally in the atmosphere as a trace gas and it can work as a greenhouse gas.",
         <http://purl.obolibrary.org/obo/IAO_0000118> "N2O",
-        rdfs:label "nitrous oxide"
+        rdfs:label "nitrous oxide",
+        
```

Comment:
I suggest that we add here CHEBI's label `dinitrogen oxide` as alternative term.

Reply:
That's a good suggestion. I added it now.